addons/cookoff/functions/fnc_cookOffBox.sqf

```diff
@@ -44,9 +44,9 @@ if (local _vehicle) then {
 
         // These functions are smart and do all the cooking off work
         if (local _box) then {
-            _box call FUNC(secondaryExplosions);
             if (_box getVariable [QGVAR(enableAmmoCookoff), GVAR(enableAmmoCookoff)]) then {
```

Comment:
Yeah on further thought I'd definitely be in favour of:

1. Removing this condition entirely
2. Changing the line 74 of `XEH_postInit.sqf` to use `getVariable [QGVAR(enableAmmoCookoff), GVAR(enableAmmobox)])`
3. Adding a second eden setting that applies to boxes and vehicles to toggle their `QGVAR(enableAmmoCookoff)`
4. Making the existing eden setting only apply to vehicles again
5. Clarifying the difference between the global settings `GVAR(enableAmmoCookoff)` and `GVAR(enableAmmobox)` (the former applied to vehicles, the latter applies to boxes specifically).

I personally think that produces the most flexible settings for mission designers to use.